### PR TITLE
Adding changelog process to Mayflower

### PR DESCRIPTION
**Pull request by mrossi113**
## Description
Adding some changelog process to the Mayflower per a discussion.

## Related Issue / Ticket

- [JIRA issue]()
- [Github issue]()

## Steps to Test

1. 

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

* 

#### @TODO

*

### CODE REVIEW

**Comment by mrossi113** [comment]
@clairesunstudio @avrilpearl add any comments or make changes!

File: changelogs/template.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Note: See http://keepachangelog.com/ for more info about changelogs.
+
+___CHANGE IMPACT___
+- Minor
+- Major
+- Patch
+
+___SEMANTIC VERSION___
+This should be done by the developer who is merging this PR into dev branch.
```
Comment (mrossi113) [inline changelogs/template.txt]: @avrilpearl @clairesunstudio do you think this should be linked to the \docs\versioning.md file that in mayflower already?

Comment (clairesunstudio) [reply changelogs/template.txt]: do you mean having a note that says: check \docs\versioning.md for more details?

Comment (mrossi113) [reply changelogs/template.txt]: @clairesunstudio yes we have this versioning.md file which includes definitions of what is a patch, minor, and major along with how semantic versioning is determined in Mayflower. `https://github.com/massgov/mayflower/blob/dev/docs/versioning.md`

**Review by clairesunstudio — APPROVED** [review]

File: changelogs/template.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+___DESCRIPTION___
+Change_type (see below)
+Change_impact (see below)
+- DP-1234: Adds apples to apple trees for admin apple pickers
+
+___SEMANTIC VERSION (see below)___
```
Comment (powpow12) [inline changelogs/template.txt:6]: @mrossi113  @clairesunstudio  what is this section supposed to be used for?

Also, do you find people are doing this right? For example, here is a changelog from dev
image

This is how I would expect it to look,
image

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
 <!-- Please use TICKET Description of ticket as PR title (i.e. DP-1234 Add back-to link on Announcement template)  -->
+Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](/docs/change-log-instructions.md)
+
```
Comment (powpow12) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: was this meant to be a hidden comment as well?

@mrossi113  ^

File: changelogs/template.txt (hunk 2 above)
Comment (mrossi113) [reply changelogs/template.txt:6]: @powpow12 the semantic version was suggested during a talk with @avrilpearl about adding the changelog to Mayflower and in the PR add this so @clairesunstudio and @avrilpearl could discuss further in detail. From further discussion I thought this was removed from the PR? @clairesunstudio

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (mrossi113) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: I did not hide this comment we can hide if necessary. @avrilpearl @clairesunstudio any input on this subject?

**Pull request merged**